src/lib.rs

```diff
@@ -580,32 +619,50 @@ impl<T> Menu<T> {
 
         let x = x.min(x_max);
 
-        let area = Rect::new(x, y, drop_down_width, group.len() as u16);
+        let area = Rect::new(x, y, min_drop_down_width, min_drop_down_height);
 
         // clamp to ensure we draw in areas
         let area = area.clamp(*buf.area());
 
         Clear.render(area, buf);
 
-        buf.set_style(area, self.drop_down_style);
-
+        buf.set_style(area, self.default_item_style);
+
+        // Render menu border
+        use ratatui::prelude::Margin;
```

Comment:
add `use ratatui::prelude::*` at the top?  If Block and Borders not imported and only used at this place, consider use fully qualified path like `rataui::widgets::Block::default()`?

Reply:
I moved them to the top where use statements usually are.